test/Libraries/RevitIntegrationTests/BugTests.cs

```diff
@@ -811,6 +811,80 @@ public void AllElementsInActiveViewReturnsViewableElements()
             Assert.AreEqual(elements.Count(), 3);
         }
 
+        [Test]
+        [Category("RegressionTests")]
+        [TestModel(@".\Bugs\EleBindingTest_MAGN-7937.rfa")]
+        public void EleBindingTest_MAGN_7937()
+        {
+
+            var model = ViewModel.Model;
+
+            string filePath = Path.Combine(workingDirectory, @".\Bugs\EleBindingTest_MAGN-7937.dyn");
+            string testPath = Path.GetFullPath(filePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+            AssertNoDummyNodes();
+
+            // check all the nodes and connectors are loaded
+            Assert.AreEqual(2, model.CurrentWorkspace.Nodes.Count());
+            Assert.AreEqual(1, model.CurrentWorkspace.Connectors.Count());
+
+            RunCurrentModel();
+
+            string refPtNodeId = "23e2f77c-bd3f-4376-83aa-45dedde795b8";
+            var refPt = GetPreviewValue(refPtNodeId) as ReferencePoint;
+            Assert.IsNotNull(refPt);
+            Assert.AreEqual(7, refPt.Z);
```

Comment:
When comparing double values, we'd better compare with a some precision. I suggest to use ShouldBeApproximately